lib/uniq/index.js

```diff
@@ -0,0 +1,7 @@
+export default function uniq(arr) {
+  return arr.reduce(function(acc, item) {
+    return acc.indexOf(item) === -1
+      ? acc.concat(item)
+      : acc
+  }, [])
```

Comment:
short but slow (connected to issue https://github.com/Kelin2025/nanoutils/issues/41)